src/notebookConverter.ts

```diff
@@ -132,8 +127,7 @@ export class NotebookConverter implements Disposable {
             // but the concat document has a bug where if all the cells are destroyed, the concat document
             // stops tracking.
             // So the alternative solution is to just regenerate on the next cell add.
-            this.activeDocuments.delete(key);
-
+            this.deleteWrapper(concatDocument);
```

Comment:
Deleting just the key from the map made events fire correctly but then cell change notifications were being sent twice. This is because just removing the item from the map didn't disconnect the old cell change event binding.